jbpm-test-coverage/src/test/java/org/jbpm/test/functional/CorrelationKeyTest.java

```diff
@@ -162,4 +168,55 @@ public void testMultiValuedKeyUniqueButInclusive() {
         ProcessInstance processInstance2 = ksession.startProcess(PROCESS, key2, null);
         assertProcessInstanceActive(processInstance2.getId());
     }
+
+    @Test
+    public void testCreateProcessInstanceDefaultValue() {
+        CorrelationKey key = keyFactory.newCorrelationKey(SIMPLE_KEY);
+
+        Map<String, Object> parameters = new HashMap<String, Object>();
+
+        WorkflowProcessInstance processInstance = (WorkflowProcessInstance) ksession.startProcess(PROCESS, key,
+                parameters);
+
+        assertEquals("defaultProc", processInstance.getVariable("procVar"));
+        assertEquals(Integer.valueOf(1), processInstance.getVariable("secondVar"));
+
+    }
+
+    @Test
+    public void testCreateProcessInstanceWithDefaultValueAndParams() {
+        CorrelationKey key = keyFactory.newCorrelationKey(SIMPLE_KEY);
+
+        Map<String, Object> parameters = new HashMap<String, Object>();
+
+        parameters.put(VARIABLE_ID, VARIABLE_VALUE);
+        WorkflowProcessInstance processInstance = (WorkflowProcessInstance) ksession.startProcess(PROCESS, key,
+                parameters);
+       
+        assertEquals(VARIABLE_VALUE, processInstance.getVariable("procVar"));
+        assertEquals(Integer.valueOf(1), processInstance.getVariable("secondVar"));
+
+    }
+
+    @Test
+    public void testCreateProcessInstanceComplexTypeDefaultValue() throws SAXException, IOException {
+
+        String address = "<org.jbpm.test.domain.Address>\n" + "          	 <street>abc</street>\n"
+                + "						   <number>29</number>\n" + "						   <city>def</city>\n"
+                + "						</org.jbpm.test.domain.Address>";
```

Comment:
I think you can write this without spaces and carriage returns (that's why xmlunit is useful)